src/node_activation.rs

```diff
@@ -282,7 +285,14 @@ pub trait ContractStakeModule {
 
         // set user stake to UnBondPeriod
         let mut stake_sent = BigUint::from(node_ids.len()) * self.node_config().get_stake_per_node();
-        self.user_data().convert_user_stake_desc(UserStakeState::PendingDeactivation, UserStakeState::UnBondPeriod, &mut stake_sent);
+        self.user_data().convert_user_stake_desc(
+            UserStakeState::PendingDeactivationFromSale,
+            UserStakeState::UnBondPeriod,
```

Comment:
We currently have an unbond queue, everyone in the unbond queue asked to unstake (came from sale), everyone else was dragged out against their will.

It might be good though to make the model more explicit, I'm thinking about it.